.github/workflows/ci.yml

```diff
@@ -189,21 +193,36 @@ jobs:
           # setting up maven often times out on macOS
           maven: ${{ matrix.os != 'macos-latest' }}
       - uses: ./.github/actions/build-zeebe
+        id: build-zeebe
         with:
           go: false
-          maven-extra-args: -am -pl qa/integration-tests -T1C
-      - name: Run smoke test
+          maven-extra-args: -T1C
+      - uses: ./.github/actions/build-docker
+        id: build-docker
+        # Currently only Linux runners support building docker images without further ado
```

Comment:
:thought_balloon: it's rare to encounter a wild "ado" :smile: I guess it's technically appropriate it here though ;)